README.md

```diff
@@ -188,6 +204,20 @@ new Date().getSeconds();
 // => 49
 new Date().getHours();
 // => 19
+
+// date-fns
+import getSeconds from "date-fns/getSeconds";
+import getHours from "date-fns/getHours";
+getSeconds(new Date());
```

Comment:
I'd like to know why using this is better than the native method.

Reply:
it's just a comparison, do we need to give the best one we think? or just let people select what they want.

Reply:
I think we should list the pros and cons.

Reply:
Otherwise, people will ask "why not X lib/module?"